bundles/org.openhab.binding.lghombot/README.md

```diff
@@ -0,0 +1,95 @@
+# LG HomBot Binding
+
+The binding integrates hacked LG HomBot VR6260 based vacuum robots.
+Details on how to modify your HomBot can be [found here](https://www.roboter-forum.com/index.php?thread/10009-lg-hombot-3-0-wlan-kamera-steuerung-per-weboberfläche/).
+The binding uses the HTTP service on port 6260 right now.
+
+Please take care when modding your HomBot! This binding is a complement to a modded HomBot not an excuse to do the modding.
+Remember, You are responsible if You brick Your HomBot.
+
+## Supported Things
+
+Hacked LG HomBot series 62XX are supported.
+
+The service that the binding connects to is actually lg.srv [found here](https://sourceforge.net/projects/lgsrv/) running on a HomBot.
+
+## Discovery
+
+The auto-discovery should hopefully find your HomBot, be aware that it will try to connect to port 6260 on all IP-addresses on your subnet.
+If you already know the IP-address of your HomBot you might as well just plug it in.
+
+## Thing Configuration
+
+The thing only requires an IP-address to function, this should hopefully be found using the discovery method.
+
+## Channels
+
+
+| Channel Type ID | Item Type | Description                                                                                   | Read/Write |
```

Comment:
Could you make it less wide, must likely your editor supports some kind of block select to help you here.

Reply:
Is 80 chars wide good? Or is there another limit?

Reply:
Sounds great, remark was mainly because it did not fit here on the github review screen while the content is not that big